### PR TITLE
Create RVRS.Load_VIP_DeathFuneralDirectorPr.sql

### PR DESCRIPTION
**Pull request by sailendrasingh1**
Code Review for Funeral Director first draft. 06/30/2023

### CODE REVIEW

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralDirectorPr.sql [hunk 1]
```diff
@@ -220,9 +220,9 @@ PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)
    
 		SELECT *
 			
-	        ,CASE WHEN FirstName like '[^a-zA-Z]%' THEN 'FirstName|Error:Not a valid First Name' ELSE '' END AS LoadNote_1
-	        ,CASE WHEN MiddleName like '[^a-zA-Z]%' THEN 'MiddleName|Error:Not a valid Middle Name' ELSE '' END AS LoadNote_2
-	        ,CASE WHEN LastName like '[^a-zA-Z]%' THEN 'LastName|Error:Not a valid Last Name' ELSE '' END AS LoadNote_3
+	        ,CASE WHEN LEFT(FirstName,1) LIKE '[^a-zA-Z]' AND FirstName NOT LIKE '-%' THEN 'FirstName|Error:FirstName does not start with Alphabet' ELSE '' END AS LoadNote_1
+	        ,CASE WHEN LEFT(MiddleName,1) LIKE '[^a-zA-Z]' and MiddleName NOT LIKE '-%' THEN 'MiddleName|Error:MiddleName does not start with Alphabet' ELSE '' END AS LoadNote_2
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralDirectorPr.sql:224]: As per the data conversion rule what you did it correct . 
I'm not sure there is similar cases in the other table but would be nice if we excluded the value that were already included in the data conversion table. For example the middle name : 
Replace (LEFT (MiddleName,1), '-','a') LIKE '[^a-zA-Z]' AND MiddleName NOT IN (<<values from data conv>>)

**Review by almabekwoldesenbet — APPROVED** [review]
Hello Mac. We need to discuss about some the suffixes that needed to be handled in the conversions(165) e.g., IIII - this should be IV

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralDirectorPr.sql (hunk 1 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralDirectorPr.sql:224]: Fixed

**Comment by sailendrasingh1** [comment]
Suffix we are skipping it for now, and deal with it later

**Review by almabekwoldesenbet — APPROVED** [review]
Looks good to me

**Pull request merged**